modules/get-started/pages/licensing/overview.adoc

```diff
@@ -14,7 +14,7 @@ Redpanda, Redpanda Console, and Redpanda Connect are available in community and
 
 [TIP]
 ====
-Go to the Redpanda Self-Managed quickstart to set up a local Redpanda cluster and try the Enterprise Edition free for up to 60 days.
+Go to the Redpanda Self-Managed quickstart to set up a local Redpanda cluster and try the Enterprise Edition free for up to 30 days.
```

Comment:
I think 'up to 60 days' is still correct because you get 30 days by default and it's also free to extend for another 30 days.